code/modules/mob/new_player/sprite_accessories_tail.dm

```diff
@@ -338,6 +338,14 @@
 	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
 	color_blend_mode = ICON_ADD
 
+/datum/sprite_accessory/tail/special/tesharitail_damaged_cenilimi_synthfeather
+	name = "Teshari tail, damaged + synthfeather Cenilimi Cybernetics prosthetic"
```

Comment:
Requesting a consistent naming scheme for all of the tesh markings in this file.
```suggestion
	name = "Teshari tail, damaged, feathered, Cenilimi Cybernetics prosthetic"
```
There _has_ to be a better way than making every single permutation because the number of qualifiers is frankly absurd at this point.